.github/workflows/mypy_primer.yml

```diff
@@ -93,12 +92,20 @@ jobs:
     name: Join artifacts
     runs-on: ubuntu-latest
     needs: [mypy_primer]
-    permissions:
-      contents: read
     steps:
       - name: Merge artifacts
         uses: actions/upload-artifact/merge@v4
         with:
           name: mypy_primer_diffs
           pattern: mypy_primer_diffs-*
           delete-merged: true
+
+  call_comment_workflow:
+    name: Call mypy_primer_comment workflow
+    needs: [join_artifacts]
+    permissions:
+      contents: read
```

Comment:
isn't this the default?

```suggestion
```

Reply:
Turned out this is required: https://github.com/python/mypy/actions/runs/12597384030

Reply:
Oh, TIL, sorry!!

Reply:
@AlexWaygood whenever `permissions:` is declared, it does not augment the global definition but replaces it entirely, FYI.